Auth0/ClaimValidators.swift

```diff
@@ -261,10 +261,15 @@ struct IDTokenAuthTimeValidator: JWTValidator {
 
     func validate(_ jwt: JWT) -> LocalizedError? {
         guard let authTime = jwt.claim(name: "auth_time").date else { return ValidationError.missingAuthTime }
+
         let currentTimeEpoch = baseTime.timeIntervalSince1970
-        let authTimeEpoch = authTime.timeIntervalSince1970 + Double(maxAge) + Double(leeway)
-        guard authTimeEpoch < currentTimeEpoch else {
-            return ValidationError.pastLastAuth(baseTime: currentTimeEpoch, lastAuthTime: authTimeEpoch)
+        let authEpoch = authTime.timeIntervalSince1970
+
+        let authAge = currentTimeEpoch - authEpoch
+        let adjustedMaxAge = Double(maxAge) + Double(leeway)
```

Comment:
Converting these separately to Double values prevents a crash when attempting to add any positive int to `.max`